RELEASE.md

```diff
@@ -0,0 +1,21 @@
+# How to release
```

Comment:
It looks like this file will be copied to many repositories. Is there a more healthy way to handle this file?
(Put master file on somewhere and mention to the file in each repository?)